sdk/appcenter-data/src/main/java/com/microsoft/appcenter/data/LocalDocumentStorage.java

```diff
@@ -96,6 +98,18 @@
     private static final String BY_PARTITION_AND_DOCUMENT_ID_WHERE_CLAUSE =
             String.format("%s = ? AND %s = ?", PARTITION_COLUMN_NAME, DOCUMENT_ID_COLUMN_NAME);
 
+    /**
```

Comment:
Can you explain the changes on that class? It's hard to read the changes and I don't understand what it's supposed to be doing now.

Reply:
these are for supporting another method called `getContentValues`, because another method need to pass everything... which is not accommodate to the situation... (for some column like download time, which requires first read the document and then write the value again to keep unchanged... which seems too redundant... :D )